help-jellyfin/index.html

```diff
@@ -0,0 +1,439 @@
+										document.getElementById("s_financier").style.display = "none";
+									}
+									if (document.getElementById("s_troubleshoot").style.display == 'none') {
+										document.getElementById("s_troubleshoot").style.display = "block";
+										document.getElementById("s_document").style.display = "none";
+										document.getElementById("s_financier").style.display = "none";
+										document.getElementById("s_troubleshoot").scrollIntoView(true);
+									} else {
+										document.getElementById("s_troubleshoot").style.display = "none";
+									}
+								}
+								document.getElementById("s_financier_button").onclick = function() {
+									if (document.getElementById("s_financier").style.display == 'block') {
+										document.getElementById("s_document").style.display = "none";
+										document.getElementById("s_troubleshoot").style.display = "none";
+									}
+									if (document.getElementById("s_financier").style.display == 'none') {
+										document.getElementById("s_financier").style.display = "block";
+										document.getElementById("s_document").style.display = "none";
+										document.getElementById("s_troubleshoot").style.display = "none";
+										document.getElementById("s_financier").scrollIntoView(true);
+									} else {
+										document.getElementById("s_financier").style.display = "none";
+									}
+								}
+							</script>
+							<div id="s_document" style="display:none;">
+								<p>Documentation is simultaneously very important, but very neglected in a lot of projects. We want to be different, and you can help! If you come across anything that you think should be documented, such as how to do things, configuration steps, or just general helpful pointers, we welcome contributions to <a href="https://github.com/jellyfin/jellyfin-docs" style="color:#fff">our Documentation Repository</a>, visible <a href="../docs/" style="color:#fff">here</a>.
+							</div>
+							<div id="s_troubleshoot" style="display:none;">
+								<p>We have a large and diverse userbase, with so many features that the combinations and configurations are almost endless. But as a volunteer-run project, the contributors can often be limited in the help they can provide. If you are well-versed in Jellyfin's operation, we welcome you to try to help troubleshoot problems your fellow users are having. Troubleshooting generally occurs in our <a href="https://matrix.to/#/#jellyfin:matrix.org" style="color:#fff">main</a> and <a href="https://matrix.to/#/#jellyfin-troubleshooting:matrix.org" style="color:#fff">troubleshooting</a> Matrix rooms, on <a href="https://reddit.com/r/jellyfin" style="color:#fff">our Reddit Subreddit</a>, and on <a href="https://forum.jellyfin.org" style="color:#fff">our Forum</a>. Hanging around those places and helping your fellow users, in a kind, courteous, and respectful manner, earns our eternal gratitude!</p>
```

Comment:
also help in Github issues should be appreciated I think

Reply:
I want to steer them away from touching issues as a complete newbie, just because then we get clutter. I'd rather people start from the Reddit/Channels first then move onto issues.